layers/+spacemacs/spacemacs-layouts/funcs.el

```diff
@@ -701,6 +701,20 @@ STATE is a window-state object as returned by `window-state-get'."
 
 ;; Eyebrowse transient state
 
+(defun spacemacs/clone-workspace ()
+  "Clone the current workspace.
+And show a minibuffer message, ex:
+Workspace: 1, cloned to Workspace: 2"
+  (interactive)
+  (let* ((eyebrowse-new-workspace nil) ; nil = clone current workspace
+         (current-workspace-nr (eyebrowse--get 'current-slot))
+         (window-configs (eyebrowse--get 'window-configs))
+         (slots (mapcar 'car window-configs))
+         (next-free-workspace-nr (eyebrowse-free-slot slots)))
+    (eyebrowse-create-window-config)
```

Comment:
By calling `eleybrowse-create-window-config`, you are repeating L711-713.
Why not to directly call `(eyebrowse-switch-to-window-config next-free-workspace-nr)` instead?

```elisp
(defun eyebrowse-create-window-config ()
  "Creates a window config at a yet unoccupied slot."
  (interactive)
  (let* ((window-configs (eyebrowse--get 'window-configs))
         (slots (mapcar 'car window-configs))
         (slot (eyebrowse-free-slot slots)))
    (eyebrowse-switch-to-window-config slot)))
```

Reply:
Fixed. Your right, thanks.